snapshot_dbg_cli/snapshot_debugger_rtdb_service.py

```diff
@@ -50,8 +51,18 @@ def get_schema_version(self):
   def set_schema_version(self, version):
     return self.rest_service.set(self.schema.get_path_schema_version(), version)
 
-  def get_debuggees(self):
-    return self.rest_service.get(self.schema.get_path_debuggees())
+  def get_debuggees(self, current_time_unix_msec):
+    debuggees = self.rest_service.get(self.schema.get_path_debuggees()) or {}
+
+    # The result will be a dictionary, convert it to an array, while also
+    # filtering out any invalid entries, normalize_debuggee returns None for
+    # invalid debuggee.
```

Comment:
There's at least one too many `,`s in here.  I think that it parses better if "normalize_debuggee ..." is the start of a new sentence.

Reply:
Good call.